input/pagecontent/index.md

```diff
@@ -1,7 +1,8 @@
 ### Introduction
 The **pregnancy passport** contains all the important medical information needed during pregnancy, at birth and for six weeks afterwards to ensure comprehensive care. This information is entered in the pregnancy passport by the treating healthcare providers, such as gynecologists, midwives and general practitioners, as well as institutions such as clinics, birthing centres, practices or midwife organizations. 
 
-The CH EPREG implementation guide describes the FHIR representation of the **e**lectronic **preg**nancy passport in Switzerland. It is dependent on [CH Core](http://fhir.ch/ig/ch-core/index.html) and [CH Term](http://fhir.ch/ig/ch-term/index.html), which describe the Swiss-specific context, particularly in relation to the Swiss Electronic Patient Record (EPR).   
+The CH EPREG implementation guide (IG) describes the FHIR representation of the **e**lectronic **preg**nancy passport in Switzerland. It is dependent on [CH Core](http://fhir.ch/ig/ch-core/index.html) and [CH Term](http://fhir.ch/ig/ch-term/index.html), which describe the Swiss-specific context, particularly in relation to the Swiss Electronic Patient Record (EPR). CH EPREG follows the [Swiss layered approach to FHIR interoperability](http://fhir.ch/ig/ch-core/usage-swiss-core-artifacts.html), which promotes the reuse of common core artifacts across different exchange formats. Like other Swiss exchange format IGs such as CH EMED (medication) and CH VACD (vaccination), CH EPREG leverages core clinical profiles where relevant to pregnancy care, ensuring consistency across Swiss healthcare documentation while applying pregnancy-specific constraints where needed.

```

Comment:
would prose to remove the sentence: , particularly in relation to the Swiss Electronic Patient Record (EPR) as CH Core is not particularly for the EPR